ACE/ace/config-integrity-common.h

```diff
@@ -0,0 +1,166 @@
+#ifndef ACE_CONFIG_INTEGRITY_COMMON_H
+#define ACE_CONFIG_INTEGRITY_COMMON_H
+
+#define ACE_INTEGRITY
+
+#define ACE_LACKS_GETPGID
+#define ACE_LACKS_SETPGID
+#define ACE_LACKS_SETREUID
+#define ACE_LACKS_SETREGID
+#define ACE_LACKS_SETSID
+#define ACE_LACKS_GETEGID
+#define ACE_LACKS_SETEGID
+#define ACE_LACKS_GETUID
+#define ACE_LACKS_SETUID
+#define ACE_LACKS_GETEUID
+#define ACE_LACKS_SETEUID
+#define ACE_LACKS_GETGID
+
+#define _REENTRANT
```

Comment:
Shouldn't this come from the OS headers?

Reply:
I'm not entirely sure how Integrity uses it, but older versions of glibc left this up to the user to define.